HHS/Head-Start-TTADP
# [TTAHUB-2507] Setup frontend for multiple regional dashboard types, add feature flag "training_report_dashboard"

## Description of change
- Add "training_report_dashboard
- For a user that doesn't have the flag/is not an admin, nothing will change
- For a user with the flag, or as an admin, the regional dashboard now has tabs at the top. I've left the vestigial "/" path there despite it not correlating to a tab link, so going to the base page means you will see tabs without an active link
- Setup routes at "/training-reports" and "/all-reports." These are empty dashboard pages.

## How to test
- Run migration.
- Confirm that for a user without flag/admin privileges, the experience on the regional dashboard is unchanged.
- Confirm that for an admin/flagged user, that the tab navigation appears on /regional-dashboard and that the two links take you to empty dashboard pages

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2507


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
